os/include/tinyara/bluetooth/bluetooth.h

```diff
@@ -40,6 +40,7 @@
  * Public Function Prototypes
  ****************************************************************************/
 
+#include <stdio.h>
```

Comment:
please include <tinyara/config.h> above stdio.h

Reply:
Done. Thank you.